templates/telemetry_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE
```

Comment:
I think it's going to be a bit tricky to actually make this query incremental. I think we can delay that, though. This `init.sql` query which creates the table from scratch isn't ridiculously expensive; it scans 2.1 TB, so costs ~$10. For now, my plan is to schedule this query to completely recreate the table every day. We can refactor later to make it more efficient if needed.

Reply:
Sounds good. The logic of the query is a bit convoluted. When I have a chance I'll think about whether there is a smarter way to write it that would make it easier to "incrementalize". 

In the abstract its a table that *should* be incremental, in the sense that a given user-service combination can only have one first date, and that date can't change. But the contortions that we have to go through to get those dates makes it hard.